utils/libmysql_client/libmysql_client.cpp

```diff
@@ -104,6 +104,7 @@ int LibMySQL::run(const char* query)
     {
         fErrStr = "fatal error runing mysql_real_query() in libmysql_client lib";
         ret = -1;
+        return ret;
```

Comment:
I don't know this code.  This has the side effect of not setting fRes; it will be left with its old value.  If that is important, then this will need to be fixed.  If it's not important, then this will work.  I'll leave it to your judgement.

Reply:
Yes, that is fine, the only possibility in that case is fRes = NULL unless we are using the class incorrectly. It is only designed to make a connection, execute a single query and then end. The destructor cleans up. If run returns a -1 then fRes won't be used because an error occurred. Everything is aborted. Exceptions would have probably been better here but this is basically a merge of two sections of code that were doing the same thing.
At a later date we should maybe consider connection pooling here for performance.

Reply:
Looks good to me then, will merge once 1.2.1 is finished.